src/main/kotlin/no/nav/syfo/personstatus/application/IPersonOversiktStatusRepository.kt

```diff
@@ -7,5 +7,7 @@ interface IPersonOversiktStatusRepository {
 
     fun updateArbeidsuforhetvurderingStatus(personident: PersonIdent, isAktivVurdering: Boolean): Result<Int>
 
+    fun upsertSenOppfolgingKandidatStatus(personident: PersonIdent, isUbehandletKandidat: Boolean): Result<Int>
```

Comment:
Ser vi har `…KandidatStatus` en del steder. Er dette nøyaktig for hva vi prater om eller snakker vi egentlig om en `SenOppfolgingKandidatVurdering` eller `SenOppfolgingVurdering` 🤔 Litt usikker selv egentlig 🧐
```suggestion
    fun upsertSenOppfolgingKandidatStatus(personident: PersonIdent, isUbehandletKandidat: Boolean): Result<Int>
```